pyperf/tests/test_timeit.py

```diff
@@ -220,6 +220,9 @@ def test_cli_snippet_error(self):
     # When the PyPy program is copied, it fails with "Library path not found"
     @unittest.skipIf(pyperf.python_implementation() == 'pypy',
                      'pypy program cannot be copied')
+    # FIXME: macOS fail this test
```

Comment:
any idea what's going on?

Reply:
Not digging yet, but soon I will find out what's going on

Reply:
I would prefer to not merge this change. Otherwise, the test may be skipped forever and nobody will notice :-(